docs/docs/_index.md

```diff
@@ -114,64 +117,90 @@ OK
+[
+	{
+		"name": "Leonard Cohen",
+		"lastSeen": 1478476800,
+		"loggedOut": true
+	}
+]
+```
+
+### Python example
 
 This code snippet shows how to use RedisJSON with raw Redis commands from Python with [redis-py](https://github.com/redis/redis-py):
 
 ```Python
 import redis
 
 data = {
-    'foo': {
-        'bar' : 'baz'
+    'dog': {
+        'scientific-name' : 'Canis familiaris'
     }
 }
 
 r = redis.Redis()
 r.json().set('doc', '$', data)
 doc = r.json().get('doc', '$')
-foo = r.json().get('doc', '$.foo')
-bar = r.json().get('doc', '$..bar')
+dog = r.json().get('doc', '$.dog')
+scientific_name = r.json().get('doc', '$..scientific-name')
 ```
 
-### Building on Ubuntu 20.04
+### Build on Ubuntu 20.04
 
-The following packages are required to successfully build on Ubuntu 20.04:
+The following packages are required to successfully build RedisJSON on Ubuntu 20.04:
 
 ```
-sudo apt install build-essential llvm cmake libclang1 libclang-dev cargo
+$ sudo apt install build-essential llvm cmake libclang1 libclang-dev cargo
 ```
-Then, run `make` or `cargo build --release` in the repository directory
+Then, run `make` or `cargo build --release` in the repository directory.
 
-### Loading the module to Redis
+### Load the module to Redis
 
 Requirements:
 
-* [Redis v6.0 or above](http://redis.io/download)
+* [Redis v6.0 or later](http://redis.io/download)
+
+To load the RedisJSON module, use one of the following methods:
+
+* [Configuration file](#configuration-file)
+* [Command-line option](#command-line-option)
+* [MODULE LOAD command](#module-load-command)
+
+#### Configuration file
 
 We recommend you have Redis load the module during startup by adding the following to your `redis.conf` file:
 
 ```
 loadmodule /path/to/module/target/release/librejson.so
 ```
 
-On Mac OS, if this module has been built as a dynamic library use:
+On Mac OS, if this module was built as a dynamic library, run:
 
 ```
 loadmodule /path/to/module/target/release/librejson.dylib
 ```
 
-In the above lines replace `/path/to/module/` with the actual path to the module's library.
+In the preceding lines, replace `/path/to/module/` with the actual path to the module's library.
+
+#### Command-line option
 
-Alternatively, you can have Redis load the module using the following command line argument syntax:
+Alternatively, you can have Redis load the module using the following command-line argument syntax:
 
 ```bash
-~/$ redis-server --loadmodule ./target/release/librejson.so
+$ redis-server --loadmodule ./target/release/librejson.so
 ```
 
-Lastly, you can also use the [`MODULE LOAD`](/commands/module-load) command. Note, however, that `MODULE LOAD` is a **dangerous command** and may be blocked/deprecated in the future due to security considerations.
+#### `MODULE LOAD` command
+
+You can also use the `MODULE LOAD` command to load RedisJSON. Note that `MODULE LOAD` is a **dangerous command** and may be blocked/deprecated in the future due to security considerations.
```

Comment:
Module or redis commands enclosed with backticks, e.g., ``` `JSON.GET` ```, are substituted with the corresponding command url, but I'm not sure if spaces are handles (converted to dashes?) - so if not, in order for it to be a clickable link, it would require reverting back to ```[`MODULE LOAD`](/commands/module-load)```

Reply:
When I look at the [current version of the doc](https://redis.io/docs/stack/json/#loading-the-module-to-redis:~:text=/commands/module%2Dload), including `(/commands/module-load)` doesn't seem to render correctly and shows `/commands/module-load` in the text itself.

Reply:
@itamarhaber I assume the rendering problem is due to the automatic substitution of commands in backticks, which occurs also within a link (inside the square brackets of the url), thus breaking the link format `[url](description)`
It seems it should skip links.